cloud/aws/aws_kinesis.cc

```diff
@@ -223,6 +211,43 @@ class KinesisController : public CloudLogControllerImpl {
   void SeekShards();
 };
 
+Status KinesisController::Initialize(CloudEnv* env) {
+  Status status = CloudLogControllerImpl::Initialize(env);
```

Comment:
i think this variable `status` conflict with function `status()`. Somehow the gcc in travis doesn't allow it. Can you change the name to `s` or `st` to avoid conflict?